### PR TITLE
Fixing the packager  infinite loop on Windows

### PR DESCRIPTION
**Pull request by mqli**
The issue here https://github.com/facebook/react-native/issues/2787

The root path '/' dosn't match windows root path

### CODE REVIEW

**Comment by mkonicek** [comment]
Thanks for the PR!

@martinbigio Can you review this please?

**Comment by martinbigio** [comment]
Looks good to me!

**Comment by martinbigio** [comment]
@facebook-github-bot shipit

File: packager/react-packager/src/DependencyResolver/DependencyGraph/ResolutionRequest.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ class ResolutionRequest {
         realModuleName => {
           const searchQueue = [];
           for (let currDir = path.dirname(fromModule.path);
-               currDir !== '/';
+               currDir !== path.parse(fromModule.path).root;
```
Comment (sebmck) [inline packager/react-packager/src/DependencyResolver/DependencyGraph/ResolutionRequest.js]: The result of `path.parse` is constant and this is being evaluated on each iteration so it makes sense to hoist this.

**Comment by mqli** [comment]
Thanks @sebmck , yet I think the other code on master causing the ci building failed.

**Comment by martinbigio** [comment]
@mqli the tests failures are unrelated. Could you update the PR with @sebmck suggestion?

**Pull request closed without merging**